acceptance-tests/requirements.txt

```diff
@@ -0,0 +1,9 @@
+papermill==2.1.3
```

Comment:
This seems like a weird collection of dependencies. If we keep them here should we add them to the dependabot config?

Reply:
Sure.